e2e/go.mod

```diff
@@ -3,11 +3,11 @@ module github.com/bloxapp/ssv/e2e
 go 1.20
 
 require (
-	github.com/attestantio/go-eth2-client v0.19.11-0.20240129201044-9d799aaab2bd
+	github.com/attestantio/go-eth2-client v0.21.9
```

Comment:
unless this version is well tested, we can already update to latest released version [21.10](https://github.com/attestantio/go-eth2-client/releases/tag/v0.21.10)

Reply:
it's been well tested :)

let's upgrade in another PR (maybe the Go 1.22 PR)